packages/compass-crud/src/components/document-list.jsx

```diff
@@ -23,6 +25,20 @@ import {
 import './index.less';
 import './ag-grid-dist.css';
 
+
+const listAndJsonStyles = css({
```

Comment:
can you take screenshots of these changes? what are those for? thanks!

Reply:
added in preview. can you check it?

Reply:
Yeah, is a bit too narrow to get an idea of all the changes, I see a lot of spaces being adjusted here and there in crud in the crud view, is all we changed already in those screenshots?

Btw if is too much work no worries I can also pull this branch and check it later!

Reply:
Ohhh sorry you added it, thanks!!